hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java

```diff
@@ -148,6 +141,21 @@
      */
     PERFORMANCE_MONITOR_HUMAN_FRIENDLY_FORMAT("hazelcast.performance.monitor.human.friendly.format", true),
 
+    PERFORMANCE_MONITOR_SLOW_OPERATIONS_PERIOD_SECONDS("hazelcast.performance.monitor.slowoperations.period.seconds", 60),
```

Comment:
You should define these properties with a `TimeUnit`, so you can benefit from the `getMillis()`, `getSeconds()` methods in `HazelcastProperties`, e.g. `PERFORMANCE_MONITOR_SLOW_OPERATIONS_PERIOD_SECONDS("hazelcast.performance.monitor.slowoperations.period.seconds", 60, SECONDS),`